contents/blog/best-uxcam-alternatives.mdx

```diff
@@ -0,0 +1,427 @@
+---
+title: 'The best UXCam alternatives & competitors, compared'
+date: 2024-07-19
+author:
+  - lior-neu-ner
+rootpage: /blog
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/posthog-alternatives/posthog-alternatives.jpeg
+featuredImageType: full
+category: General
+tags:
+  - Comparisons
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+## 1. PostHog
+
+- **Founded:** 2020
+- **Similar to:** Fullstory, LogRocket
+- **Typical users:** Engineers and product teams
+
+![PostHog](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/blog/best-pendo-alternatives/posthog.png)
+
+### What is PostHog?
+
+[PostHog](/) (that's us 👋) is an open-source platform that combines [product analytics](/product-analytics), [session replay](/session-replay), [A/B testing](/ab-testing), and [feature flags](/feature-flags). It also has a [data warehouse for startups](/docs/data-warehouse) and a [customer data platform (CDP)](/docs/cdp). This means it's not only an alternative to UXCam, but also tools like [Statsig](/blog/posthog-vs-statsig) and [LaunchDarkly](/blog/posthog-vs-launchdarkly).
+
+By combining all these tools into one platform, it eliminates the need for stitching together integrations between third-party tools, and makes it easier for engineers to work with data. PostHog is popular with engineering-led companies, like AI startup [ElevenLabs](/customers/elevenlabs) and [carVertical](/feature-flags), which use PostHog for both session replays and analytics.
+
+#### Key features
+
+- 📈 **Product analytics:** Custom trends, funnels, user paths, retention analysis, and segment user cohorts. Also, direct [SQL querying](/docs/product-analytics/sql) for power users.
+
+- 📺 **Session replays:** View exactly how users are using your app. Includes event timelines, console logs, network activity, and 90-day data retention.
```

Comment:
@marandaneto do we support all these feature for mobile session replay?

Reply:
yes